### ISSUE TITLE
[18Rhl] Implement KS rules for Rheingold Express (8-train) as variant?

### ISSUE THREAD

**Issue by kodemaniak** [tobymao/18xx#11425]
**What's your idea?**

18Rhl currently seems to implement older rules. Specifically the 8-train ("Rheingold Express") seems to work differently in the current rules (i.e. the rules from the Kickstarter). In the 18xx.games implementation it may skip cities, which is a rule that does not seem to exist in the KS rules.

**Are there related ideas?**

Only found an older BGG discussion about the Rheingold Express: https://boardgamegeek.com/thread/2650757/how-do-the-8rge-trains-work. But it is not explicitly about skipping cities.

**Comment by perwestling** [comment]
Yes, 18Rhl at 18xx.games was implemented before the Kickstarter campaign was started. So it would need a refresh to comply with the new version. Most likely where would be a 18Rhl version 2 for it.

That said, I am not sure the rules really differ for the 8 train. My interpretation to treat it as an 8E train could possibly fit both. 
I might have to ask the designer about this.

**Comment by perwestling** [comment]
I have recently started to work on this - took only a year!

As it looks 18 Rhineland (2024) will be a subgame of 18 Rhl, so it will appear as a variant box.
If clicking that it will be possible to also chose some of the new variants in 18 Rhineland (2024), and any variant only available in original 18 Rhl will disappear as choice.

Also the 18 LRA variant will be another subgame of 18 Rhl, using the same principles.

Implementation has started, but do not expect anything in alpha during first half of 2026, maybe during second half - we'll see.

**Comment by perwestling** [comment]
This is covered by #11672 so closing this as a separate issue.
